src/test/java/seedu/application/logic/commands/RemindCommandStub.java

```diff
@@ -20,13 +19,9 @@ public class RemindCommandStub extends Command {
 
     public static final String SHOWING_REMIND_MESSAGE = "Opened reminder window.";
 
-    private static final UpcomingInterviewPredicateStub UPCOMING_INTERVIEW_PREDICATE =
-            new UpcomingInterviewPredicateStub();
-
```

Comment:
ah so then there's no longer a need for `RemindCommandStub` since the only difference between this stub and `RemindCommand` is that this uses the `UpcomingInterviewPredicateStub`, which is no longer the case?

Reply:
That's a good point. Actually it is a bit odd to be stubbing the class to be tested in the first place. Usually stubs are for replacing the dependencies of the class to be tested. Maybe after this PR is merged you can help replace both stubs with the original classes if you'd like? You can design the test cases to be based off the current date and time so that they always pass/fail the predicate no matter when they are run

Reply:
yepp makes sense I'll do that after this is merged~